### ISSUE TITLE
Searching in the left pane of vscode.dev does not work for non-main/non-default branches

### ISSUE THREAD

**Issue by bekatd** [microsoft/vscode#139063]
Searching in the left pane of vscode.dev does not work for non-main/non-default branches.

Steps to reproduce:
- in vscode.dev select any non-main/non-default branch
- search for something that is specific for this branch
- results are not displayed
- now search for something that is in the main branch (while still being in the previously selected branch)
- the results shown in the left pane are from the default branch (which we did not select) 

**in other words, the search always returns results from the default main branch, even though you have selected a different branch** 

![aaaaaaaaaaaaa](https://user-images.githubusercontent.com/6948919/145957919-fca0b576-d614-4666-a2de-f8c08ba281d3.PNG)

Version: 1.63.0
Commit: 7db1a2b88f7557e0a43fec75b6ba7e50b3e9f77e
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:95.0) Gecko/20100101 Firefox/95.0
Embedder: vscode.dev

**Comment by JacksonKearl** [comment]
Thanks for reporting this. It's a known issue that the search only applies to the main branch, but I'd expect you to see some messages informing you of such and asking if you'd like to instead search by downloading a copy of the repo: 
![image](https://user-images.githubusercontent.com/8586769/146285598-3c907204-d2bd-480b-add8-7dd6f097e2c9.png)

Have you seen anything like this? Do you have an example repo I could test with?

**Comment by bekatd** [comment]
I have not seen any messages asking me to download a copy of the repo, but I remember seeing a warning message that search is failing/unable without index.

Sorry I don't have any test repos. I encountered this issue on a standard repo

**Comment by JacksonKearl** [comment]
New version of remotehub is out that will seach in indicies by default
